be/src/storage/tablet_updates.cpp

```diff
@@ -4011,7 +4013,7 @@ Status TabletUpdates::get_column_values(const std::vector<uint32_t>& column_ids,
                                         vector<std::unique_ptr<Column>>* columns, void* state) {
```

Comment:
I think we should first get `unique_id` of `column_ids` and use unique id to get column values. 
e.g.
If the initial schema is `k1, k2, v1, v2, v3` and after some alter operation the schema is `k1, k2, v2, v4, v3`. Then we do partial update and the partial columns are `k1, k2, v2`, we need to get the column value of `v4` and `v3`. And the column id is 3 and 4. But I think we may get old value from column v2 and column v3 because column 3 and column 4 in old rowset are column  `v2` and column `v3`.

Reply:
I would fix this issue in next PR(#27645)

Reply:
I would fix this issue in next PR